oak_loader/Cargo.toml

```diff
@@ -21,16 +21,22 @@ default = ["oak_debug"]
 anyhow = "*"
 env_logger = "*"
 hex = "*"
+http = "*"
 log = "*"
 oak_abi = { path = "../oak_abi" }
 oak_runtime = { path = "../oak_runtime", default-features = false }
+oak_proxy_attestation = { path = "../experimental/proxy_attestation" }
```

Comment:
I think it would be a good idea to add an "oak_experimental" feature that is not enabled by default so that the oak server only depends on experimental crates (and their dependencies) if it is explicitly requested to be an experimental build.

Reply:
Added an `oak_attestation` feature, and it's running as part of the `--server-variant=experimental` which is now default in CI.